sweble-wikitext-components-parent/swc-engine/src/main/java/org/sweble/wikitext/engine/config/WikiConfigImpl.java

```diff
@@ -383,7 +383,7 @@ public void addI18nAlias(I18nAliasImpl alias)
 				throw new IllegalArgumentException("This alias (`" + alias.getId() + "') registeres the name `" + a + "' twice.");
 
 			if (old2 != null)
-				throw new IllegalArgumentException("The name `" + a + "' was already registered by the alias `" + old2.getId() + "' when trying to register it for alias `" + alias.getId() + "'.");
+				System.out.println("The name `" + a + "' was already registered by the alias `" + old2.getId() + "' but we are also registering it for `" + alias.getId() + "'.");
 
 			nameToAliasMap.put(lcAlias, alias);
```

Comment:
@tgalery I think the changes of this PR won't actually allow for multiple aliases, as `put` will overwrite existing Map entries here. So this code is relying on the "last" alias that has been put to the map. 

I guess, you should change the Map to contain a `Collection<String>` as value here? Thereby, one could associate multiple aliases under one `lcAlias`.

Could you check `WikiConfigImpl` again for my thoughts?

Reply:
Will do @mawiesne I had a similar worry myself but hadn't had time to check things through.

Reply:
see: https://docs.oracle.com/javase/10/docs/api/java/util/Map.html#put(K,V)

> If the map previously contained a mapping for the key, the old value is replaced by the specified value.

Reply:
I meant more in the context of the overall application, it seems that two maps are updated in this function and since I don't know the code well I'm unsure what they are meant to model. That's why I was nudging the maintainers. But you are defo right in the sense that the current solution might have undesired side effects, specially wrt to the second map update. Specially [here](https://github.com/sweble/sweble-wikitext/blob/dd21ad2e870e3691d6e816023afeb143f89d6e21/sweble-wikitext-components-parent/swc-engine/src/main/java/org/sweble/wikitext/engine/config/WikiConfigImpl.java#L391)